CHANGELOG.md

```diff
@@ -0,0 +1,13 @@
+# Change Log
```

Comment:
decided not to go with github releases?

Reply:
No I did not. Should we not use CHANGELOG.md if we use github release? How does it work?

Reply:
well I guess we don't need a markdown changelog if we are going with github. Or we could track  "latest changes" in the repo while the full log is on the releases (as in allensdk).

Reply:
I have updated the guidelines, stating that release notes are pulled from the changelog which in turn must be updated by committers.